decidim-core/app/commands/decidim/create_report.rb

```diff
@@ -76,7 +76,11 @@ def send_hide_notification_to_moderators
       participatory_space_moderators.each do |moderator|
         next unless moderator.email_on_moderations
 
-        ReportedMailer.hide(moderator, @report).deliver_later
+        if hidden_by_admin?
+          ReportedMailer.hidden_by_admin(moderator, @report, current_user).deliver_later
```

Comment:
I'd also change the method `hidden_by_admin?` so it is `hidden_manually?` (as a moderator can also hide resources, but they're not an "admin" as such)

```suggestion
          ReportedMailer.hidden_manually(moderator, @report, current_user).deliver_later
```

Reply:
Suggestion applied in 8555f7f9dd . Done without Github's help as we needed to rename a file.